packages/label-sync/src/label-sync.ts

```diff
@@ -42,11 +42,25 @@ interface Repos {
 // from the local copy.  We are using the `PushEvent` to detect the change,
 // meaning the file running in cloud will be older than the one on master.
 let labelsCache: Labels;
-async function getLabels(github: GitHubAPI, repoPath: string) {
+async function getLabels(github: GitHubAPI, repoPath: string): Promise<Labels> {
   if (!labelsCache) {
     await refreshLabels(github, repoPath);
   }
-  return labelsCache;
+  const labels = {
+    labels: labelsCache.labels.slice(0),
```

Comment:
What's the purpose of a `labels.slice(0)`?

Reply:
this creates a copy of the `labels` so that we don't end up populating additional labels in the cache each time we access labels.

creating a cache means that the labels are cached between invocations of cloud functions.